src/st/st-widget.c

```diff
@@ -1617,7 +1695,7 @@ st_widget_ensure_style (StWidget *widget)
   g_return_if_fail (ST_IS_WIDGET (widget));
 
   if (widget->priv->is_style_dirty)
-    st_widget_recompute_style (widget, NULL);
+      st_widget_recompute_style (widget, NULL);
```

Comment:
We use two spaces after a bracket-less if statement.

Reply:
OK